collectors/cronjob.go

```diff
@@ -120,8 +153,12 @@ func (jc *cronJobCollector) collectCronJob(ch chan<- prometheus.Metric, j v2batc
 		addGauge(descCronJobSpecStartingDeadlineSeconds, float64(*j.Spec.StartingDeadlineSeconds))
 	}
 
-	addGauge(descCronJobInfo, 1, j.Spec.Schedule, string(j.Spec.ConcurrencyPolicy))
+	// If the cron job is suspended, don't track delays
+	if j.Status.LastScheduleTime != nil && !*j.Spec.Suspend {
```

Comment:
agreed! `CronJob` [only has](https://github.com/kubernetes/kubernetes/blob/60604f8/staging/src/k8s.io/api/batch/v2alpha1/types.go#L54-L148) a `LastScheduleTime` timestamp right now though (there's no `creationTime` to say when the job was created) so I don't see how to accomplish that with the current API